website/sidebars.json

```diff
@@ -2,15 +2,15 @@
   "docs": {
     "OVERVIEW": [
       "overview/who-is-this-for",
-      "overview/what-is-near"
+      "overview/what-is-near",
+      "local-setup/create-account"
```

Comment:
i would move the `create-account` section to the first item in the `QUICK START` section